### ISSUE TITLE
Reactivity pattern not allowed

### ISSUE THREAD

**Issue by philippe-wm** [solidjs-community/eslint-plugin-solid#32]
**Your Question**
How can we make the 2nd pattern valid wrt reactivity rule?

```typescript
// OK
function createBar(v: Accessor<unknown>): void {...}
createBar(() => bar());

// KO
function createFoo(v: { onBar: Accessor<unknown> }): void {...}
createFoo({ onBar: () => bar() });
```

**Comment by joshwilsonvu** [comment]
Thanks for the question. Since the reactivity rule can't know the implementation of your custom hooks, it should be permissive about calling it with signals. Currently it looks for arguments that are inline functions and allows reactivity there, but it misses anything else like in your case.

I think I'll call this a bug, because your code looks perfectly valid--the rule needs to be more flexible around custom hooks. Look for a fix in the next release!

**Comment by joshwilsonvu** [comment]
The warnings you're seeing should be taken care of in `v0.7.4`. Thanks again!
